.changelog/1708.txt

```diff
@@ -0,0 +1,4 @@
+```release-note:bug
+core/runner: prevent use of operation flags on `runner agent` command
```

Comment:
Just `cli:` here is fine. We have to document these (I think on the list for the "0.5 cycle") but we try to make these more user-actionable rather than map directly to our package structure.